pkg/satellite/users.go

```diff
@@ -32,6 +32,11 @@ type UserInfo struct {
 	Password  string `json:"password"`
 }
 
+// IsValid checks UserInfo validity and returns error describing whats wrong
+func (i *UserInfo) IsValid() error {
```

Comment:
Avoid single letter receivers. They are harder to grasp especially when moving from one package to another. In this case it could be `user`.